.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,20 @@
+name: docker-build
+
+on:
+  push:
```

Comment:
Should have `on: pr` too

Reply:
Updated.

Reply:
LGTM, thanks! https://github.com/jarrettj/laminas-mvc-skeleton/runs/6225033761?check_suite_focus=true#step:3:755